src/ngraph/op/pad.cpp

```diff
@@ -174,3 +174,97 @@ std::shared_ptr<Node> op::Pad::get_default_value() const
     }
     return std::make_shared<op::Broadcast>(input_value(1), get_shape(), axes);
 }
+
+const string op::v1::Pad::type_name{"Pad"};
+
+op::v1::Pad::Pad(const Output<Node>& arg,
```

Comment:
The spec doesn't say so, but for other ops things like `pads_begin` and `pads_end` are required to be `int64`. Should we do that here?

In any case, if `pads_begin` and `pads_end` are constant, we should still have the checks from the `v0` op for negative dim after padding, and for the input dimensions being too small in the EDGE or REFLECT modes.